scheduler/core/plans/__init__.py

```diff
@@ -102,8 +101,9 @@ def nir_slots(self,
 
     @staticmethod
     def time2slots(time_slot_length: timedelta, time: timedelta) -> int:
```

Comment:
this is scattered around the code the same logic

Reply:
I'm not quite sure what you mean... do you mean we have this logic implemented several times (in which case, we should conglomerate them)? I see a lot of calls to this method (which should probably be a function in lucupy) but I don't off the top of my head see it implemented more than once?

Reply:
Oh, I just did a search for `ceil`, and now I see it all over. I'll spend a bit of time cleaning that up today.